static/src/javascripts/projects/common/modules/commercial/track-ad.js

```diff
@@ -0,0 +1,30 @@
+define([
+    'Promise',
+    'common/utils/mediator'
+], function(Promise, mediator) {
+    return trackAd;
+
+    function trackAd(id) {
+        return new Promise(function (resolve, reject) {
+            var onAdLoaded = function (event) {
+                if (event.slot.getSlotElementId() === id) {
```

Comment:
maybe we could have mediator emit an event specific to the slot. So instead of listening to `modules:commercial:dfp:rendered`, we listen to `modules:commercial:dfp:rendered:slotId`